README.md

```diff
@@ -1,109 +1,178 @@
-# DataStax Node.js Driver for Apache Cassandra Quickstart
+# DataStax NodeJS Quickstart using Netflix
```

Comment:
Should we use the same title format as the Java sample?

"DataStax Desktop - Node.js Netflix example"